src/pages/management/configuration-api/v2.0/index.mdx

```diff
@@ -2562,6 +2563,119 @@ rules[0][operator]=contains"
 
 <Text>
 
+### Ads traffic
+
+The greeting will be sent when the customer first visit url contains specified ads parameters.
```

Comment:
Here the same case as above
1. The greeting will be sent when the customer's first visited URL contains specified ads parameters.
2. The greeting will be sent when the customer first visits the URL containing specified ads parameters.